compliance/cis_k8s/test_data.rego

```diff
@@ -17,7 +17,10 @@ filesystem_input(filename, mode, uid, gid) = {
 # genrates `process` type input data
 process_input(process_name, arguments) = {
 	"type": "process",
-	"resource": {"command": concat(" ", array.concat([process_name], arguments))},
+	"resource": {
+		"command": concat(" ", arguments),
```

Comment:
```suggestion
		"command": concat(" ", array.concat([process_name], arguments)),
```
I wouldn't change this one, even though it's not accurate, we still want the first field not as a parameter no?

Reply:
Yeah - nice catch